get-started/codelab/index.md

```diff
@@ -319,106 +283,114 @@ class is, itself, immutable, but the State class persists over the
 lifetime of the widget.
 
 In this step, you’ll add a stateful widget, RandomWords, which creates
-its State class, RandomWordsState. The State class will eventually
-maintain the proposed and favorite word pairs for the widget.
+its State class, RandomWordsState. You'll then use RandomWords as
```

Comment:
Actually, this is one place where we differ from Google style guide. We don't always codefont classes, as it can make the text harder to read.

Reply:
SGTM. My 2 cents: it can make the text harder to read, but it also makes it easier to find parts of the prose that are referring to code.

When _do_ we codefont class names then? (You can answer in another thread if you prefer :)